src/index.js

```diff
@@ -1,189 +0,0 @@
-const p = require('path')
-const fs = require('fs')
-// const printAST = require('ast-pretty-print')
-const {getReplacement, requireFromString} = require('./helpers')
-
-module.exports = prevalPlugin
-
-function prevalPlugin(babel) {
-  const {types: t, template, transformFromAst} = babel
-  const assignmentBuilder = template('const NAME = VALUE')
-  return {
-    name: 'preval',
-    visitor: {
-      Program(path, {file: {opts: fileOpts}}) {
-        const firstNode = path.node.body[0] || {}
-        const comments = firstNode.leadingComments || []
-        const isPreval = comments.some(isPrevalComment)
-
-        if (!isPreval) {
-          return
-        }
-
-        comments.find(isPrevalComment).value = ' this file was prevaled'
-
-        const {code: string} = transformFromAst(
-          path.node,
-          null,
-          /* istanbul ignore next (babel 6 vs babel 7 check) */
-          /^6\./.test(babel.version)
-            ? {}
-            : {
-                babelrc: false,
-                configFile: false,
-              },
-        )
```

Comment:
I think this logic was changed 🤔

Reply:
It has. I was struggling with they types for the old call and took inspiration from `babel-plugin-codegen`. I don't know enough about Babel plugins or ASTs to tell you if it's an important change or not, but all the tests still passed do ran with it.

Happy to wrestle with the types more if you think it's important?

Reply:
I'd rather `// @ts-expect-error` than change the logic here I think.

Reply:
No worries, will put it back

Reply:
This has been done.  Please not that the regex has been updated to a `startsWith` as per the `eslint` rules